tests/appsec/test_asm_standalone.py

```diff
@@ -37,11 +37,15 @@ def setup_no_appsec_upstream__no_attack__is_kept_with_priority_1__from_minus_1(s
 
     def test_no_appsec_upstream__no_attack__is_kept_with_priority_1__from_minus_1(self):
         spans_checked = 0
-        for data, _, span in interfaces.library.get_spans(request=self.r):
+        for data, trace, span in interfaces.library.get_spans(request=self.r):
             if not REQUESTDOWNSTREAM_RESOURCE_PATTERN.search(span["resource"]):
                 continue
 
-            assert span["metrics"]["_sampling_priority_v1"] < 2
+            if span["metrics"].get("_sampling_priority_v1") is None:
+                assert trace[0]["metrics"]["_sampling_priority_v1"] < 2
+            else:
+                assert span["metrics"]["_sampling_priority_v1"] < 2
+
```

Comment:
so, in java `span != trace[0]`?

i'm wondering if 
  `assert trace[0]["metrics"]["_sampling_priority_v1"] < 2`
would be sufficient in all the tracers because in node works

Reply:
My main concern is if this is true for all the languages, but let me change it and let's see

Reply:
are the passing for python too, no?

Reply:
Seems that works fine for all the tracers.. but it will be nice to check it in tracers code